apps/site/pages/en/about/get-involved/index.md

```diff
@@ -9,6 +9,7 @@ If you are interested in getting involved with the Node.js community, there are
 
 ## Community Discussion
 
+- The [Node.js official Discord server](https://discord.gg/nodejs) is a place to chat with other Node.js developers and getting official news from the Node.js project.
```

Comment:
This doesn't look right:
![CleanShot 2025-02-11 at 23 23 52@2x](https://github.com/user-attachments/assets/4c195272-f5ba-47a2-9889-54cbdd4b7e8f)

Reply:
Yes, this link is "fake" let me replace into a REPLACEME link for now.